code/pizzabot.js

```diff
@@ -4,4 +4,49 @@ const pepperoni = "Pepperoni Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+console.log(vegetarian)
+console.log(hawaiian)
+console.log(pepperoni)
+console.log(pizzaPrice)
+
+function greeting() {
```

Comment:
Writing out a function like this is fine! But writing it as `const greeting = () => {}` is the new way of writing it (ES6). It is good that you know that and try to use it because that's probably how you will create functions in you work